tests/unit/mixins/component-test.js

```diff
@@ -28,3 +36,102 @@ test( 'Assert is thrown when isViewable() is not implemented on the derived clas
         'Assertion was thrown'
     );
 });
+
+test( 'showContent() computed property throws assertion when `possible` property is not a boolean', function( assert ) {
+    const subject = AugmentedObject.create();
+    const callShowContent = () => subject.get( 'showContent' );
+
+    // Array
+    subject.set( 'possible', [] );
+
+    assert.throws(
+        callShowContent,
+        'The `possible` property is an Array'
+    );
+
+    // Boolean
+    subject.set( 'possible', false );
+
+    let assertionThrown = false;
```

Comment:
I put it in for completeness, but we are testing the `showContent` computed property return values in the tests on lines 117 to 137 so those would error out anyway so I guess it is not really necessary.